backend/app/repository/gitea_provider.py

```
@@ -0,0 +1,821 @@
+                            git_domain=git_domain,
+                            type=self.type,
+                            private=repo.get("private", False),
+                        ).model_dump()
+                        for repo in paginated_repos
+                    ]
+                )
+                continue
+
+            try:
+                headers = self._build_headers(git_token)
+                response = requests.get(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)